File: Day1/ifelse.py
# def compareifelse(a:int,b:int):
#     if (a>b):
#         print("Greater")
#     elif (a<b):
#         print("Smaller")
#     else:
#         print("equal")
    
def compareifelse(a:int,b:int):
    if (a>b):
        return "Greater"
    elif (a<b):
        return "Smaller"
    else:
        return "equal" 
a = input()
b=input()
print(compareifelse(a,b))

# if __name__=="__main__":
#     a = input()
#     b=input()
#     print(compareifelse(a,b))

File: Day2/ifelse.c
#include <stdio.h>

int main()
{
    int a,b;
    printf("Enter two numbers\n");
    scanf("%d",&a);
    scanf("%d",&b);
    if (a>b)
    {
        printf("Greater");
    }
    else if (a<b)
    {
        printf("Smaller");
    }
    else 
    {
        printf("Equal");
    }
    return 0;
}